modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage by using the {oc-first}.
+
+.Prerequisites
+
+* You have installed the latest version of `oc`. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
```

Comment:
I don't see this "_CLI tools_" section - am I missing it or is this maybe an older reference?
```suggestion
* You have installed the latest version of `oc`. For more information, see "Getting started with the OpenShift CLI".
```

Reply:
Thank you for letting me know.
Looks like they introduced new cards as an extra nav level in the docs.redhat.com.
I've updated it to in `_Configure: CLI tools_`.